### ISSUE TITLE
[A11y] Desktop Work Show Page-CPD Parent: Universal Viewer Gallery View 

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1978]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Parent: https://staging.oregondigital.org/concern/generics/df727084j?locale=en

When I tab to the Universal Viewer and click on the gallery view icon and view the thumbnails. I can use the zoom in and zoom out. However, I am unable to tab into the actual image thumbnails.

When I am on the plus icon and tab next it skips the thumbnail images and goes directly to the UV search box.

However, if I use the index tab, I am able to tab through the list and select an image.

### Suggested Fix

Be able to select a image thumbnail under the thumbnails tab.

### Can anybody QA or does this need an expert?

Anybody

**Comment by azle** [comment]
Updated 12/30/2024

I can move through the tab index and thumbnails by using tab. Then use control+option+space to enter tab content.

I can tab through list in index however in thumbnails content area. I can tab to the minus and plus but unable to get to the thumbnails. After I tab to the plus icon the tab next goes to the embed icon on the bottom left of the viewer.

**Comment by azle** [comment]
Updated 12/30/2024

@jsimic  I have been checking links by removing the staging portion of the link. However is this right. Why does this viewer have just the thumbnails in the tab content area, its a different view from above:

https://oregondigital.org/concern/generics/df727084j?locale=en

We should be consistent.
